node_package/tests/react_on_rails.spec.js

```diff
@@ -0,0 +1,12 @@
+var test = require('tape');
```

Comment:
Check out this repo for the start: https://github.com/shakacode/js-promises-testing